qvm/sys-gui.sls

```diff
@@ -29,6 +29,14 @@ service:
 
 {{ load(defaults) }}
 
+# dom0 is GuiVM for sys-gui
+dom0-guivm-gui-agent:
+  qvm.vm:
+    - name: dom0
+    - service:
+      - enable:
+        - guivm-gui-agent
+
```

Comment:
There is no gui-agent in dom0, this looks suspicious. And especially after https://github.com/QubesOS/qubes-core-admin-client/pull/118/commits/27fee07b3839b8320e858fb63ce479940ba8e36a, looks unnecessary.

Reply:
This is deprecated. It was an attempt to make qvm-start-daemon starting when I noticed (reason of the quoted commit) there were no /var/run/qubes-service/ file.